roles/openssl_cert_gen/templates/xdmod.conf

```diff
@@ -0,0 +1,50 @@
+<IfModule !mod_ssl.c>
+    LoadModule ssl_module modules/mod_ssl.so
+</IfModule>
```

Comment:
This module check can be removed ? Comments ??

Reply:
done